src/ControllerV2/BeneficiaryResetPasswordController.php

```diff
@@ -33,8 +33,8 @@ public function choice(Beneficiaire $beneficiary): Response
     #[Route(path: '/{id<\d+>}/reset-password/email', name: 'reset_password_beneficiary_email', methods: ['GET'])]
     public function resetEmail(Request $request, Beneficiaire $beneficiary, ResettingService $service): Response
     {
-        if ($email = $beneficiary->getUser()->getEmail()) {
-            $service->processSendingPasswordResetEmail($email, $request->getLocale());
+        if ($beneficiary->getUser()->getEmail()) {
+            $service->processSendingBeneficiaryPasswordResetEmail($beneficiary);
```

Comment:
I wonder why this method takes a beneficiary, it should take a user, and be callable for any user